src/app/app.module.ts

```diff
@@ -8,6 +8,7 @@ import { HttpModule, XHRBackend, RequestOptions } from '@angular/http';
 // 3rd party modules
 import {
     BsDropdownModule,
+    CollapseModule,
```

Comment:
can this instead just be pulled in in `sidebar.module.ts`?

Reply:
There is no `sidebar.module.ts`

Reply:
d'oh i got confused, thinking we modularize this project like Temperate